tests/server/configs/sjrk.storyTelling.server.testServerWithBaseTheme.json5

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "sjrk.storyTelling.server.testServerWithCustomTheme",
+    "options": {
+        "gradeNames": ["sjrk.storyTelling.server"],
+        "components": {
+            "server": {
+                "options": {
+                    "globalConfig": {
+                        "binaryUploadDirectory": "./tests/server/uploads",
+                        "uploadedFilesHandlerPath": "./tests/server/uploads"
+                    },
+                    "components": {
+                        "storyDataSource": {
```

Comment:
This subcomponent doesn't seem to be used in the test too. It would be less confusing to only define the necessary bits that have been used by the test.

This comment applies to other config files too. Thanks.